src/core/loopback-setup.c

```diff
@@ -26,10 +26,33 @@
 #include "missing.h"
 #include "netlink-util.h"
 
-static int start_loopback(sd_netlink *rtnl) {
+struct state {
+        unsigned n_messages;
+        int rcode;
+};
+
+static int start_loopback_handler(sd_netlink *rtnl, sd_netlink_message *m, void *userdata) {
+        struct state *s = userdata;
+
+        assert(s);
+        assert(s->n_messages > 0);
+        s->n_messages--;
+
+        errno = 0;
+        log_debug_errno(sd_netlink_message_get_errno(m), "Got start error code: %m");
```

Comment:
Oh, this is ugly. `log_internal_realmv` uses the specifier error, unless it's 0, and then it uses `errno`. We should fix that, and make the log functions uses the specified code unconditionally, but that's a separate issue. This code is correct, as things are now. When that happens, also invert the order of lines 42 and 44, to avoid calling `sd_n_m_get_errno` twice.

"Got start error code: Success" looks a bit dumb, but OK.